pkg/datastore/firestore/iterator.go

```diff
@@ -59,7 +59,7 @@ func (it *Iterator) Cursor() (string, error) {
 
 	lastObjData := it.last.Data()
 
-	cursor := make(map[string]interface{}, len(it.orders))
+	cursor := make(map[string]interface{})
```

Comment:
Why this was removed?

Reply:
I want to address it by another PR which refine Firestore iterator 👀 This PR should only change things related to MySQL as it's name 🙏

Reply:
That is better. 👍